docs/tutorials/gluon/gluon_from_experiment_to_deploymen.md

```diff
@@ -0,0 +1,400 @@
+    transforms.ToTensor(),
+    transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
+])
+
+transform_test = transforms.Compose([
+    transforms.Resize(256),
+    transforms.CenterCrop(224),
+    transforms.ToTensor(),
+    transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
+])
+
+
+path = './data'
+train_path = os.path.join(path, 'train')
+val_path = os.path.join(path, 'valid')
+test_path = os.path.join(path, 'test')
+
+train_data = gluon.data.DataLoader(
+    gluon.data.vision.ImageFolderDataset(train_path).transform_first(transform_train),
+    batch_size=batch_size, shuffle=True, num_workers=num_workers)
+
+val_data = gluon.data.DataLoader(
+    gluon.data.vision.ImageFolderDataset(val_path).transform_first(transform_test),
+    batch_size=batch_size, shuffle=False, num_workers = num_workers)
+
+test_data = gluon.data.DataLoader(
+    gluon.data.vision.ImageFolderDataset(test_path).transform_first(transform_test),
+    batch_size=batch_size, shuffle=False, num_workers = num_workers)
+ ```
+
+
+### Loading pre-trained model
+
+We will use pre-trained ResNet50_v2 model, all you need to do is re-define the last softmax layer for your case. Specify the number of classes in your data and initialize the weights.
+You can also add layers to the network according to your needs.
+
+Before we go to training, one important part is to hybridize your model, it will convert your imperative code to mxnet symbolic graph. It's much more efficient to train a symbolic model,
+and you can also serialize and save the network archietecure and parameters for inference.
+
+```python
+model_name = 'ResNet50_v2'
+finetune_net = get_model(model_name, pretrained=True)
+with finetune_net.name_scope():
+    finetune_net.output = nn.Dense(classes)
+finetune_net.output.initialize(init.Xavier(), ctx = ctx)
+finetune_net.collect_params().reset_ctx(ctx)
+finetune_net.hybridize()
+
+trainer = gluon.Trainer(finetune_net.collect_params(), 'sgd', {
+                        'learning_rate': lr, 'momentum': momentum, 'wd': wd})
+metric = mx.metric.Accuracy()
+L = gluon.loss.SoftmaxCrossEntropyLoss()
+```
+
+### Fine-tuning model on your custom dataset
+
+Now let's define the test metrics and start fine-tuning.
+
+```python
+def test(net, val_data, ctx):
+    metric = mx.metric.Accuracy()
+    for i, batch in enumerate(val_data):
+        data = gluon.utils.split_and_load(batch[0], ctx_list=ctx, batch_axis=0, even_split=False)
+        label = gluon.utils.split_and_load(batch[1], ctx_list=ctx, batch_axis=0, even_split=False)
+        outputs = [net(X) for X in data]
+        metric.update(label, outputs)
+
+    return metric.get()
+
+
+lr_counter = 0
+num_batch = len(train_data)
+
+for epoch in range(epochs):
+    if epoch == lr_steps[lr_counter]:
+        trainer.set_learning_rate(trainer.learning_rate*lr_factor)
+        lr_counter += 1
+
+    tic = time.time()
+    train_loss = 0
+    metric.reset()
+
+    for i, batch in enumerate(train_data):
```

Comment:
please use the following notation:
`for i, (data, label) in enumerate(dataloader):`